components/home/Dashboard.tsx

```diff
@@ -83,44 +62,143 @@ export default function Dashboard() {
       </View>
     </View>
   );
+};
+interface Transaction {
+  amount: string;
+  category: string;
+}
+const calculateCO2E = (transactionHistory = [], amountPerKg = 0.41) => {
+  const sumDeposits = transactionHistory.reduce((acc, item: Transaction) => {
+    return item?.category.toLocaleLowerCase() === 'deposit'
```

Comment:
We have an enum for transaction categories in 'app/types/Transaction.ts' for string type safety, I think better to import  and use this:

```suggestion
    return item?.category === Category.DEPOSIT
```